tests/executors/test_base.py

```diff
@@ -0,0 +1,15 @@
+"""General mirakuru library tests."""
+
+
+def test_importing_mirakuru():
+    """Test if all most commonly used classes are imported by default."""
+    from mirakuru import *  # noqa
```

Comment:
from what I see that kind of import doesn't work for py3 ;) should be probably on module level